tools/travis/.travis.yml

```diff
@@ -26,9 +27,11 @@ matrix:
       env: SYMFONY_VERSION=3.0.*
     - php: 5.6
       env: SYMFONY_VERSION=3.1.*@dev
+    - env: SYMFONY_DEPRECATIONS_HELPER
```

Comment:
I think you have to specify a PHP version. Otherwise this env will create a job for each version.